l10n_eu_product_adr/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "ADR Products",
     "summary": "Allows to set appropriate danger class and components",
-    "version": "13.0.1.0.1",
+    "version": "13.0.1.0.2",
```

Comment:
You should update the version to `13.0.2.0.0` I think as there is a data migration
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers